code/area.dm

```diff
@@ -2821,11 +2821,11 @@ area/station/security/visitation
 
 /area/station/turret_protected/Entered(O)
 	..()
-	if (isliving(O))
-		if(!issilicon(O))
-			if (motioncamera)
-				motioncamera.newTarget(O)
-			popUpTurrets()
+	if (!isliving(O) || issilicon(O) || isintangible(O))
```

Comment:
This will conflict with `if (istype(O,/obj/blob))` below, you'll need to move that to be above this.

Reply:
Oh, durr... Let me fix that.

Reply:
Linter's gonna yell at it but shouldn't be an issue.